env.example

```diff
@@ -1,3 +1,8 @@
 export LOCATION=eastus
 
+export METRICS_SOCKET=/tmp/aro-metrics.socket
```

Comment:
can we get down to 0 configuration for dev?
e.g. hard-code the metrics socket to mdm_statsd.socket (in prod, we'll have Docker mount it in place)
and avoid requirement for the others to be set.